Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/ui/cards/BottomInfoVaccinationCard.kt

```diff
@@ -0,0 +1,30 @@
+package de.rki.coronawarnapp.vaccination.ui.cards
+
+import android.view.ViewGroup
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.databinding.VaccinationBottomInfoCardBinding
+import de.rki.coronawarnapp.greencertificate.ui.certificates.CertificatesAdapter
+import de.rki.coronawarnapp.greencertificate.ui.certificates.items.CertificatesItem
+import de.rki.coronawarnapp.util.lists.diffutil.HasPayloadDiffer
+
+class BottomInfoVaccinationCard(parent: ViewGroup) :
+    CertificatesAdapter.CertificatesItemVH<BottomInfoVaccinationCard.Item, VaccinationBottomInfoCardBinding>(
+        R.layout.dashed_line_container_layout,
+        parent
+    ) {
+
+    override val viewBinding = lazy {
+        VaccinationBottomInfoCardBinding.inflate(layoutInflater, itemView.findViewById(R.id.container_layout), true)
+    }
+
+    override val onBindData: VaccinationBottomInfoCardBinding.(
+        item: Item,
+        payloads: List<Any>
+    ) -> Unit = { _, _ -> }
+
+    object Item : CertificatesItem, HasPayloadDiffer {
```

Comment:
What is the idea behind this?

Reply:
`BottomInfoVaccinationCard` displays information about the digital covid-19 certificate when no certificate is registered. I'm reusing cards from the home screen so using the same implementation with adapter and item view holder. And if you actually talk about `HasPayloadDiffer` for an object, that's leftover from the previous implementation.
It should be updated now

Reply:
all right